prometheus/aws-ecs/image/prometheus.yml

```diff
@@ -1,13 +1,24 @@
 scrape_configs:
   - job_name: 'node-exporter'
-    scrape_interval: 5s
+    scrape_interval: <SCRAPE-INTERVAL>
     static_configs:
       - targets: ['172.17.0.1:9100'] # target is the host
     relabel_configs:
       - target_label: instance
         replacement: '<HOSTNAME>'
+      - target_label: cluster
+        replacement: '<CLUSTERNAME>'
+      - target_label: host_public_ip
+        replacement: '<HOSTPUBLICIP>'
+  - job_name: 'cAdvisor'
+    scrape_interval: <SCRAPE-INTERVAL>
+    static_configs:
+      - targets: ['172.17.0.1:9101'] # target is the host
```

Comment:
Why are you using this IP? is it true for all customers?

Reply:
yes, this is the internal ip docker bridge mode use. 172.17.0.1 refers to the host all the time.